### ISSUE TITLE
X42 invalid character

### ISSUE THREAD

**Issue by kolya182** [bitcoinjs/bip44-constants#21]
Encounter this issue with X42, @fanatid do you know what can be wrong with it ?.

Chrome console
<img width="537" alt="screenshot 2018-09-18 16 06 39" src="https://user-images.githubusercontent.com/3277207/45721408-dd5c5000-bb5c-11e8-9c10-837fb43acdac.png">

VSC
<img width="264" alt="screenshot 2018-09-18 12 10 05" src="https://user-images.githubusercontent.com/3277207/45710718-48495f00-bb3c-11e8-8f8d-4cfe0027d3d7.png">

**Comment by dcousens** [comment]
Lovely.

``` js
> Buffer.from('0x80067932', 'utf8')
<Buffer 30 78 38 30 30 e2 80 ad 36 37 39 33 32 e2 80 ac>

> Buffer.from('0x80067932', 'utf8')
<Buffer 30 78 38 30 30 36 37 39 33 32>
```

**Comment by dcousens** [comment]
It appears there is an injected unicode character in that... (or two)

**Comment by dcousens** [comment]
@fanatid where did you copy it from?

**Comment by kolya182** [comment]
@dcousens https://github.com/bitcoinjs/bip44-constants/blob/6dced9b0615d516f3f9fdcec86c604fad51441c0/update.sh#L7

**Comment by dcousens** [comment]
It is in the SLIP too

**Comment by dcousens** [comment]
https://github.com/satoshilabs/slips/pull/335

**Comment by kolya182** [comment]
https://github.com/satoshilabs/slips/pull/369

**Comment by fanatid** [comment]
fixed in 5.2.1
